girder/models/folder.py

```diff
@@ -309,7 +309,6 @@ def remove(self, folder, progress=None, **kwargs):
             if progress:
                 progress.update(increment=1, message='Deleted item ' +
                                 item['name'])
-        items.close()
```

Comment:
The cursors will each hold a read lock on the accessed elements of their collections.  This will block a write operation on those collections from completing until the function ends and the cursors are released.  Closing them is better.  Any time we do a significant amount of processing after finishing with a cursor, we should close it rather than waiting for it to be closed when it goes out of scope.